crates/router/src/connector/utils.rs

```diff
@@ -273,6 +274,13 @@ impl CardData for api::Card {
             year.peek()
         )
     }
+    fn get_card_expiry_as_mmyy(&self) -> Secret<String> {
+        let exp_year_with_2_digit = self.get_card_expiry_year_2_digit();
+        let expiry_year = exp_year_with_2_digit.peek().clone();
+        let expiry_month = self.card_exp_month.peek().clone();
+        let expiry = format!("{}{}", expiry_month, expiry_year);
+        Secret::new(expiry)
+    }
```

Comment:
remove this method use `get_card_expiry_month_year_2_digit_with_delimiter ` with empty string as delimiter